Packs/ExpanseV2/ReleaseNotes/1_0_1.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Expanse v2
+- **Breaking Change** - The **expanse-get-iprange**, **cidr** , **expanse-get-domain**, **domain**, **expanse-get-certificate**, **certificate**, **expanse-get-associated-domains**, **ip**  commands were changed to return multiple entries (entry per indicator) instead of a single entry.
```

Comment:
Not sure if we need to mention that is a breaking change: this Pack has been merged a few hours ago and is not in marketplace yet, so these commands are definitely not being used by custom scripts.

Reply:
Iill add that this will affect custom scripts only.